functional_tests/scylla_operator/test_functional.py

```diff
@@ -356,13 +356,18 @@ def test_scylla_operator_pods(db_cluster: ScyllaPodCluster):
     assert not not_running_pods, f'There are pods in state other than running: {not_running_pods}'
 
 
-def test_startup_probe_in_scylla_pods(db_cluster: ScyllaPodCluster):
-    pods = db_cluster.k8s_cluster.kubectl('get pods -A -l "app.kubernetes.io/name=scylla" -o yaml')
+def test_startup_probe_exists_in_scylla_pods(db_cluster: ScyllaPodCluster):
+    pods = get_pods_without_probe(
+        db_cluster, "startupProbe", "app.kubernetes.io/name=scylla", "scylla")
+    assert not pods, f"startupProbe is not found in the following pods: {pods}"
 
```

Comment:
@vponomaryov  I would guess that absense of other probes is also a problem, isn't it ? should be add them ?

Reply:
It is not a problem. It is optional. For the moment "readinessProbe" is the only probe in the manager pod. And the covered operator issue is exactly about it.